packages/web-components/src/components/content-item-horizontal/__stories__/README.stories.mdx

```diff
@@ -118,6 +118,34 @@ import '@carbon/ibmdotcom-web-components/es/components/content-item-horizontal/c
 </dds-content-item-horizontal-media>
 ```
 
+## With Thumbnail
+
+### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/content-item-horizontal/index.js';
+import '@carbon/ibmdotcom-web-components/es/components/content-item-horizontal/content-item-horizontal-thumbnail';
```

Comment:
since you've already import `content-item-horizontal-thumbnail` in `content-item-horizontal/index.js`, I don't think we need to add this import statement again in the readme.